translate/tools/test_podebug.py

```diff
@@ -28,8 +28,8 @@ class TestPODebug:
     debug = podebug.podebug()
 
     def setup_method(self, method):
-        self.postore = po.pofile(PO_DOC)
-        self.xliffstore = xliff.xlifffile(XLIFF_DOC)
+        self.postore = po.pofile(PO_DOC.encode())
```

Comment:
Encode to which coding? I suppose it is `utf-8` but would be great if it is explicit.